src/features/positions/stores/transform/index.ts

```diff
@@ -245,7 +230,7 @@ function transformBorrows(tokens: PositionToken[], item: PortfolioItem, position
   return transformUnderlyingAssets(tokens, currency).map(token => ({
     asset: token.asset,
     quantity: token.quantity,
-    value: getNativeValue(item.stats?.debtValue || '0', currency),
+    value: token.value,
```

Comment:
Had missed cases where there is more than 1 asset in a position (i.e. deposit) but it's not an LP positions, so the display value was duplicated as the total instead of individual